views/info/filters.pug

```diff
@@ -200,3 +220,37 @@ block content
               tr
                 td(scope="col") #[code a:reb]
                 td(scope="col") All cards illustrated by artists with "reb" in their name.
+      .card
+        #collapsePrice-syntax.card-header(data-toggle="collapse" data-target="#collapsePrice" aria-expanded="true" aria-controls="collapsePrice")
+          button(class="btn btn-link" type="button")
+            h5 Price
+        #collapsePrice.collapse(aria-labelledby="collapsePrice-syntax" data-parent="#syntax-accordion")
+          p You can use #[code price:] or #[code priceFoil:] to filter cards by price.
+          p #[strong Examples:]
+            table.table
+              tr
+                td(scope="col") #[code price>10.5]
+                td(scope="col") All non-foil cards with a price over $10.50.
```

Comment:
Price uses the price of the specific printing in the cube including foil status

Reply:
Cool, I'll update the description and example.

```suggestion
                td(scope="col") All cards in a cube whose specified printing has a price over $10.50.
```